src/pydvl/value/shapley/montecarlo.py

```diff
@@ -68,14 +63,149 @@ class MonteCarloResults(NamedTuple):
 ]
 
 
+class TruncationPolicy(abc.ABC):
```

Comment:
I will move this and everything for TMCS to another module after this PR is merged